components/hero/index.tsx

```
@@ -1,34 +1,94 @@
+            case "logo":
+                position = componentConfig.logoPosition;
+                component = <LogoCard key="logo" />;
+                break;
+            default:
+                return; // Skip if type is not recognized
+        }
+
+        if (position !== undefined) {
+            overlays[position] = component;
+        }
+    });
```

Biome (1.9.4)

[error] Prefer for...of instead of forEach.

forEach may lead to performance issues when working with large arrays. When combined with functions like filter or map, this causes multiple iterations over the same type.

(lint/complexity/noForEach)